browser/src/core/Socket.js

```diff
@@ -1400,6 +1400,7 @@ app.definitions.Socket = L.Class.extend({
 			this._map.uiManager.closeAll();
 			this._map.setPermission(app.file.permission);
 			window.migrating = false;
+			window.keyboard.hintOnscreenKeyboard(window.keyboard.onscreenKeyboardHint);
```

Comment:
This is calc only as I read in the core commit message. This code is applied to all modules.
Maybe better to do this in CalcTileLayer ?  In onStatus handling also.